core/modules/filters.js

```diff
@@ -161,6 +162,20 @@ exports.filterTiddlers = function(filterString,widget,source) {
 };
 
 /*
+Evaluates a condition and adds an item to the result-set of a filter operation depending on negation
+	title: tiddler title
+	results: the results array
+	operator: the filter operator
+	condition: the condition depending on which to add to the results (boolean)
+Returns boolean, whether or not to add an item
+*/
+exports.addResult = function(title,results,operator,condition) {
```

Comment:
I'm not sure that this should really be a method of the wiki object. It doesn't use the state of the wiki object. I guess maybe it's better in the `$tw.utils` namespace.